Podfile.lock

```diff
@@ -294,4 +294,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: 8e7f8bcf9d6e3303a7ceda8c2769276f25d90782
 
-COCOAPODS: 1.11.2
+COCOAPODS: 1.10.1
```

Comment:
@Minhdn1 : Upgrade pod to version 1.11.2
Do not using pod update

Reply:
fix complete